test/e2e-common.sh

```diff
@@ -191,6 +199,23 @@ function install_kourier() {
   kubectl apply -f ${INSTALL_KOURIER_YAML} || return 1
 }
 
+function install_ambassador() {
+  AMBASSADOR_MANIFESTS_PATH="./third_party/ambassador-0.86.1/"
```

Comment:
Will it be better to follow the naming convention and change the folder name to `ambassador-latest`?

Reply:
Fixed in next commit. Thanks!